### PR TITLE
Drop support for an empty `tl_content.ptable` column

### PR DESCRIPTION
**Pull request by leofeyer**
@contao/developers What do you think about adding a migration that automatically changes empty columns to `tl_article`?

```sql
UPDATE tl_content SET ptable='tl_article' WHERE ptable='';
```

I‘m not sure if that would be too destructive. On the other hand, an empty column is treated as `tl_article` in Contao 4, so the value cannot really have a different meaning, can it?

### CODE REVIEW

**Comment by fritzmg** [comment]
I am in favour of migrating empty to `tl_article`.

**Comment by Toflar** [comment]
Yes, migration!

**Review by m-vo — DISMISSED** [review]
LGTM, but I'd also add a migration.

**Review by ausi — DISMISSED** [review]

**Comment by leofeyer** [comment]
Migration added in f9a9e21fe118768055802f07e17f55708e68afae.

**Review by m-vo — DISMISSED** [review]

**Review by fritzmg — DISMISSED** [review]

File: core-bundle/src/Migration/Version500/EmptyPtableMigration.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            return false;
+        }
+
+        $test = $this->connection->fetchOne("SELECT TRUE FROM tl_content WHERE ptable='' LIMIT 1");
+
+        if (false !== $test) {
+            return true;
+        }
+
+        return false;
```
Comment (ausi) [inline core-bundle/src/Migration/Version500/EmptyPtableMigration.php]:
```suggestion
        return false !== $test;
```

**Review by ausi — DISMISSED** [review]
Just a nitpick 🙈

**Review by m-vo — APPROVED** [review]

**Pull request merged**